src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -152,10 +163,11 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
     nrNum: string, businessId: string, phone: string, email: string
   ): Promise<NameRequestIF> {
     const nameRequest = await LegalServices.fetchValidContactNr(nrNum, phone, email)
-    if (!nameRequest) throw new Error('Error fetching Name Request')
+    if (!nameRequest) throw new Error('Unable to fetch Name Request.')
 
-    // validateNameRequest() already throws printable errors
-    return this.validateNameRequest(nameRequest, NrRequestActionCodes.AMALGAMATE)
+    // validateNameRequest() throws printable errors
+    return this.validateNameRequest(nameRequest, NrRequestActionCodes.AMALGAMATE, null, null,
+      null, this.getEntityType)
```

Comment:
This was a bug -- the validation function didn't check that entity type matched.